lib/call.ts

```diff
@@ -26,6 +26,9 @@ class CallTranspiler extends base.TranspilerBase {
           this.emit('new');
         }
         var newExpr = <ts.NewExpression>node;
+        if (newExpr.arguments === undefined) {
```

Comment:
I think your request is for better error handling, but this silently accepts invalid TS syntax. I think it should just throw with a better error if that's helpful.